yt/docs/ru/user-guide/query-tracker.md

```diff
@@ -0,0 +1,158 @@
+# Query tracker
+
+Query tracker позволяет:
+
++ Отправлять произвольные запросы на исполнение.
++ Отслеживать исполнение.
++ Сохранять результаты.
++ Смотреть историю запросов.
+
+Запросы характеризуются:
+
++ Движком.
++ Текстом.
+
+Движок запроса контролирует исполнение запроса.
+
+Текст запроса специфичен для движка.
+
+## Движки
+
+На текущий момент поддерживаются следующие движки исполнения:
+
++ YT QL
+  + Простой движок, поддерживающий простые запросы вида `* from [//home/user/table]`
```

Comment:
You can add this link here. 
https://ytsaurus.tech/docs/ru/user-guide/dynamic-tables/dyn-query-language